src/Microsoft.TestPlatform.ObjectModel/Framework.cs

```diff
@@ -3,14 +3,592 @@
 
```

Comment:
If I am not mistaken, this file is auto-generated. Maybe we should add an auto-gen header?

Reply:
It's not autogenerated. Just part of it if generated from something else, but it won't automatically update.